cli/src/cli.rs

```diff
@@ -2209,6 +2209,16 @@ pub(crate) fn build_balance_message(
 }
 
 pub fn app<'ab, 'v>(name: &str, about: &'ab str, version: &'v str) -> App<'ab, 'v> {
+    macro_rules! ACCOUNT_STRING {
+        () => {
+            r#"Can be:
+      * a bs58 pubkey string
+      * path to keypair file
+      * '-' to pass from stdin
```

Comment:
If there are no consumers of the `-` option, I'd prefer we remove support for that.  While `-` is the appropriate way to say "get it from stdin", most commands need multiple keys, so pretty weird to offer that syntax but it'd likely only work for one pubkey.  Also, your comment doesn't say if stdin is base58 or binary. I'd guess the former.

Reply:
Worse.  It takes a keypair file on `stdin`, so JSON.  It _does_ enable encrypted keypair files, but yes only really functional in single role scenarios

Reply:
Okay, can keep it, but gotta document it

Reply:
Added the json-encoded keypair text.